### ISSUE TITLE
Add switch options to EX1 and EX2 (Horus X10)

### ISSUE THREAD

**Issue by PD5DJ** [opentx/opentx#8376]
I Like to request switch options to EX1 & EX2 on the Horus X10

The only possible selection for EX1 & EX2 now are:

- POT
- POT with detent
- MULTIPOS

The inputs can be used for switches too.

Adding the following options would be a great added value to these inputs

- TOGGLE (external pullup resistor needed)
- 2 POS
- MULTIPOS (user defined positions)

https://hobby4life.nl/?p=504

**Comment by kilrah** [comment]
Multipos includes 2-pos.

**Comment by Squirrel05** [comment]
Yes, so we can do a multiple pos switch.

**Comment by PD5DJ** [comment]
> Multipos includes 2-pos.

I dont think you understand what I mean.

Like a 2 position switch **EX1↑,EX1↓,EX2↑ and EX2↓**

EX1 and EX2 now are not selectable as this as behaviour.

**Comment by kilrah** [comment]
When you set EX1 to multipos and calibrate your 2 positions you'll have EX11, EX12, !EX11, !EX12. Same thing.

**Comment by PD5DJ** [comment]
I have tried that.. but it wont give me these switches.

I got EX1 and **EX1**
same for EX2

Also they dont act like switches. (They dont do anything)
Something is not working properly.

During calibration when setting EXT1 or EXT2 to multipos.. they show up as sliders.. no Multiposition like the 6P.

And then again It would be odd if it where only a 6 pos calibration.. better is an option to select user defined positions.
